frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-inventory-card.tsx

```diff
@@ -36,11 +32,23 @@ export const VMInventoryCard: React.FC<VMInventoryCardProps> = () => {
 
   const basePath = resourcePath(getVMLikeModel(vmiLike).kind, name, namespace);
   const DisksTitle = React.useCallback(
-    ({ children }) => <Link to={`${basePath}/${VM_DETAIL_DISKS_HREF}`}>{children}</Link>,
+    ({ children }) => (
+      <Link
+        to={`${basePath}/${VM_DETAIL_DISKS_HREF}?rowFilter-disk-types=${DiskType.DISK.getValue()}`}
```

Comment:
please mention in the bug that the issue was resolved by filtering and not by changing the count

Reply:
done :-)